.github/workflows/lint.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
           pre-commit install-hooks --config .github/pre-commit-config.yaml
 
       - name: Run the check (${{ matrix.check }})
-        run: pre-commit run --hook-stage manual ${{ matrix.check }} --all-files --config .github/pre-commit-config.yaml
+        run: pre-commit run --show-diff-on-failure --hook-stage manual ${{ matrix.check }} --all-files --config .github/pre-commit-config.yaml
```

Comment:
Would this be useful locally as well, or just in the action?

Reply:
I don't think it's useful locally as a default option; if pre-commit modifies files the developer can just review the changes, there's no need to also show a commit. It's sometimes nice to manually add this option locally in case unstaged changes conflict with changes by pre-commit.